releng/org.eclipse.xtext.generator.feature/feature.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="org.eclipse.xtext.generator"
+      label="%featureName"
+      version="2.27.0.qualifier"
+      provider-name="%providerName"
+      license-feature="org.eclipse.license"
+      license-feature-version="0.0.0">
+
+   <description>
+      %description
+   </description>
+
+   <copyright>
+      Copyright (c) 2009, 2022 itemis AG (http://www.itemis.eu) and others.
```

Comment:
This is initially added now, so just `Copyright (c) 2022 itemis AG`

Reply:
done